templates/extract_bin_file.ipynb

```diff
@@ -1,43 +1,101 @@
 {
```

Comment:
<p>This can all get extracted and put in a function. In addition, you should run the new <code>find_moly_fovs</code> function, and remove any FOVs that are moly FOVs from the list. </p><p> </p><p>In addition, add a print statement that notifies the user about any moly FOVs that are being skipped, or previously extracted FOVs that are being skipped</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>